AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -1058,6 +1060,7 @@ def mesh(property: str = 'name') -> str:
           - 'Minimum Screen Coverage' portion of the screen at which the mesh is culled; 0 (never culled) to 1
           - 'Quality Decay Rate' rate at which the mesh degrades; 0 (never) to 1 (lowest quality imediately)
           - 'Lod Override' which specific LOD to always use; default or other named LOD
+          - 'Vertex Count LOD0' the vertices in LOD 0 of the model
```

Comment:
is this a read only property? sounds like it. again this is mesh component is this supposed to be in this review of directional light?